apps/src/gamelab/animationListModule.js

```diff
@@ -24,8 +24,6 @@ export const ADD_ANIMATION = 'AnimationList/ADD_ANIMATION';
 export const ADD_ANIMATION_AT = 'AnimationList/ADD_ANIMATION_AT';
 // Args: {AnimationKey} key, {AnimationProps} props
 export const EDIT_ANIMATION = 'AnimationList/EDIT_ANIMATION';
-// Args: {AnimationKey} key
-const INVALIDATE_ANIMATION = 'AnimationList/INVALIDATE_ANIMATION';
```

Comment:
The invalidate action becomes unnecessary, because I was only using it to force animations just added from the gallery to get saved to the animation API.  Now that we have proper by-reference animations, we don't want to do that - we just keep loading them from the gallery until they are edited.